disco/disco_test.go

```diff
@@ -77,8 +77,8 @@ func TestMarshalAndParse(t *testing.T) {
 			}
 
 			gotHex := fmt.Sprintf("% x", got)
-			if gotHex != tt.want {
-				t.Fatalf("wrong marshal\n got: %s\nwant: %s\n", gotHex, tt.want)
+			if !strings.HasPrefix(gotHex, tt.want) {
```

Comment:
If we expect the end of the ping payload to be a bunch of 00s we should probably test that, either by changing the `want` for each of the ping/pong cases to be the full value or by some other method

Reply:
We don't care that it's zeros _per se_, but I'll add a check that it's the padded length.